source/slang/slang-serialize.cpp

```diff
@@ -210,10 +210,11 @@ SerialClasses::SerialClasses():
 
 // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! SerialWriter  !!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-SerialWriter::SerialWriter(SerialClasses* classes, SerialFilter* filter) :
+SerialWriter::SerialWriter(SerialClasses* classes, SerialFilter* filter, Flags flags) :
     m_arena(2048),
     m_classes(classes),
-    m_filter(filter)
+    m_filter(filter),
+    m_flags(flags)
```

Comment:
Coding style: the diffs here are an example of why it is beneficial to rearrange the punctuation for C++ constructor initializer lists, so that the punctuation comes before each initializer:

```
SerialWriter::SerialWriter(...)
    : m_arena(2048)
    , m_classes(classes)
    , m_filter(filter)
    , m_flags(flags)
{ ... }
```

Note that when the initializer list is written this way, the addition of `m_flags(...)` is a one-line diff that does *not* affect the line for `m_filter(...)` (which is unrelated to the change).